bpf/headers/cgroup.h

```diff
@@ -45,6 +45,10 @@ static inline int get_current_cgroup_info(void *ctx,
             // not in mesh
             _default.is_in_mesh = 0;
             debugf("can not get port listen for cgroup(%ld)", cgroup_id);
+            // FIXME(kebe7jun) find a better ways to avoid affecting the
+            // performance of network connections in non-Sidecar mode.
+            *cg_info = _default;
```

Comment:
I think we don't need this assignment?

Reply:
This is mainly because the hope is to be able to initialize `cg_info`, or use `memset`?

Reply:
OK, just a nit
Now I'm wondering how the e2e test failed.. it's weird
```bash
Connected to helloworld (127.128.0.1) port 5000 (#0)
```
Seems it _only_ rewrites the dst ip, how could that happen?